tests/containers/skopeo.pm

```diff
@@ -0,0 +1,119 @@
+# SUSE's openQA tests
+#
+# Copyright 2023-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: skopeo
+# Summary: Test basic skopeo commands.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';    # used in select_serial_terminal
+use utils 'zypper_call';    # used in zypper_call
+use version_utils 'package_version_cmp';
+use version_utils 'is_transactional';
+use transactional;
+
+sub run {
+    my ($self, $args) = @_;
+
+    # Ensure skopeo is installed
+    # people make a $packages array which is populated with the packages that
+    # they want installed.
+
+
+    ######### Set required variables
+
+    # Required packages
+    my $packages = 'skopeo jq';    # and add whatever else should be missing with push function
+
+    select_serial_terminal();    # Select most suitable text console
+
+    # Set a variable for my remote image
+    my $remote_image = 'docker://registry.suse.com/bci/bci-busybox:latest';
```

Comment:
```suggestion
    my $remote_image = 'registry.suse.com/bci/bci-busybox:latest';
```
It's common to refer to images by their URI only, so I would move the protocol part (`docker://`) to the actual call.